web/modules/custom/dpl_loans/src/Plugin/Block/LoanListBlock.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\dpl_loans\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\dpl_react_apps\Controller\DplReactAppsController;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides user loans list.
+ *
+ * @Block(
+ *   id = "dpl_loans_list_block",
+ *   admin_label = "List user loans"
+ * )
+ */
+class LoanListBlock extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Drupal config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  private ConfigFactoryInterface $configFactory;
+
+  /**
+   * LoanListBlock constructor.
+   *
+   * @param array $configuration
```

Comment:
You need to specify `mixed[]` as type here in order to make PhpStan quiet.
You could argue that it is not the right way to go.
But I think that is a separate discussion and my main concern is to eliminate PHPstan errors.